src/platform/logging/types.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/* eslint-disable @typescript-eslint/ban-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+export enum LogLevel {
+    // Larger numbers are higher priority.
+    Error = 40,
+    Warn = 30,
+    Info = 20,
+    Debug = 10,
+    Trace = 5,
+    Off = 0
+}
+
+export type Arguments = unknown[];
+
+export interface ILogging {
```

Comment:
Seems like it should be called ILogger. Is this just what Python named it?

Reply:
Yeah I just copied it. I agree. I'll change the name.